chromium_src/chrome/browser/component_updater/file_type_policies_component_installer.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#define RegisterFileTypePoliciesComponent RegisterFileTypePoliciesComponent_ChromiumImpl  // NOLINT
+#include "../../../../../chrome/browser/component_updater/file_type_policies_component_installer.cc"  // NOLINT
+#undef RegisterFileTypePoliciesComponent
+
+#include "chrome/browser/component_updater/component_updater_utils.h"
+#include "components/component_updater/component_updater_service.h"
+#include "extensions/common/constants.h"
+
+namespace component_updater {
+
+void OnFileTypePoliciesRegistered() {
+  component_updater::BraveOnDemandUpdate(file_type_policies_extension_id);
```

Comment:
why this is needed?

Reply:
This list of file extensions is used to determine whether or not to ping the Google download reputation service.

Reply:
so we just want to speedup the process of getting it?

Reply:
Yes. In the past, Google had to update it frequently as malicious people were finding ways around it.

Reply:
Why would we have to update it more often than Chrome does by default? In general I think OnDemandUpdate should be reserved for new component installs. If the goal is to check for updates more frequently then this probably won't accomplish that for many users who rarely close their browser (like me)

Reply:
actually this is correct, sorry